build/npm/postinstall.js

```diff
@@ -23,9 +23,17 @@ function yarnInstall(location, opts) {
 	const args = original.filter(arg => arg === '--ignore-optional' || arg === '--frozen-lockfile');
 
 	console.log(`Installing dependencies in ${location}...`);
+	const oldConfigArch = process.env['npm_config_arch']
+	if (location.startsWith("remote") && oldConfigArch == "arm64") {
+		// Temporarily set remote arch to x86, because remote portion builds
+		// against node, which is not consistently supported on Windows on Arm
+		// (yet).
+		process.env["npm_config_arch"] = "ia32"
+	}
 	console.log(`$ yarn ${args.join(' ')}`);
 	const result = cp.spawnSync(yarn, args, opts);
-
+	// Restore the architecture, if temporarily swapped.
+	process.env["npm_config_arch"] = oldConfigArch
```

Comment:
I think the build error may come from here. Probably should strengthen the check so it fires only on Windows.